integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -0,0 +1,143 @@
+# Default values for alluxio.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+
+image: alluxio/alluxio
+imageTag: 2.1.0-SNAPSHOT
+imagePullPolicy: Always
+
+# Set the tieredstore
+tieredstore:
+  levels:
+# How to use multiple layers
+# level 0 must be MEM, and it's better to point to /dev/shm
+#  - alias: MEM
+#    level: 0
+#    path: /dev/shm
+#    #type: emptyDir
+#    type: hostPath
+#    quota: 10GB
+#    high: 0.95
+#    low: 0.7
+#  - alias: SSD
+#    level: 1
+## The path is inside the container
+#    path: /alluxio-ssd
+## the volume type, it may be hostPath, emptyDir
+#    type: hostPath
+#    quota: 30GB
+#    high: 0.9
+#    low: 0.7
+
+# How to use single layer
+  - alias: MEM
+    mediumtype: MEM,SSD
+    level: 0
+    path: /dev/shm,/alluxio-ssd
+    type: hostPath
+    quota: 10GB,50GB
+    high: 0.95
+    low: 0.7
+
+
+
```

Comment:
nit: be consistent w/ spacing. either use a single or 2 newlines between sections